src/main/java/somethingrandom/dataaccess/local/FileUserDataAccessObject.java

```diff
@@ -0,0 +1,76 @@
+package somethingrandom.dataaccess.local;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import somethingrandom.entity.Item;
+import somethingrandom.entity.ReferenceItem;
+import somethingrandom.entity.DelayedItem;
+import somethingrandom.entity.ActionableItem;
+import somethingrandom.usecase.DataAccessException;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+public class FileUserDataAccessObject {
+    private final File dataFile;
+    private final Map<UUID, Map<String, String>> items;
+
+    public FileUserDataAccessObject(String dataFilePath) throws DataAccessException {
+        dataFile = new File(dataFilePath);
+        items = new HashMap<>();
+        save();
```

Comment:
(a) not a fan of doing I/O in the constructor

(b) ...won't this just empty everything in the file?